packages/Interface/src/OperatorVector/DTK_IntegrationPointSet.cpp

```diff
@@ -180,18 +180,16 @@ void IntegrationPointSet::finalize()
 {
     // Build a globally contiguous ordering of point global ids.
     EntityId num_local_ip = d_points.size();
-    EntityId num_global_ip = 0;
-    Teuchos::reduceAll( *d_comm, Teuchos::REDUCE_SUM,
-			num_local_ip, Teuchos::ptrFromRef(num_global_ip) );
+
+    EntityId invalid = Teuchos::OrdinalTraits<Tpetra::global_size_t>::invalid();
     Teuchos::RCP<const Tpetra::Map<int,EntityId> > map =
-	Tpetra::createContigMap<int,EntityId>(num_global_ip,num_local_ip,d_comm);
+        Tpetra::createContigMap<int,EntityId>(invalid, num_local_ip, d_comm);
```

Comment:
[`Tpetra::createContigMap`](https://trilinos.org/docs/dev/packages/tpetra/doc/html/classTpetra_1_1Map.html#a98d53a205cee366b65fc4bc8f3c1bd06) is not very well documented but looking at this [`Tpetra::Map`](
https://trilinos.org/docs/dev/packages/tpetra/doc/html/classTpetra_1_1Map.html#a5eec5a0f6b1d716a260f6922afaa7d0c) constructor, it seems `Tpetra` will compute the global number for us.